cvat-sdk/cvat_sdk/core/downloading.py

```diff
@@ -58,6 +61,17 @@ def download_file(
             except ValueError:
                 file_size = None
 
+            if output_path.is_dir():
+                content_disposition = next(
+                    part.strip()
+                    for part in response.headers.get("Content-Disposition", "").split(";")
+                    if part.strip().startswith("filename=")
```

Comment:
This should be matched case-insensitively to conform to the spec.

Reply:
`headers` is a special [dict that handles this](https://github.com/urllib3/urllib3/blob/b8f512b225c99c552f461c5a8ed250652c41ee93/src/urllib3/_collections.py#L204).

Reply:
I'm not talking about getting the header, I'm talking about the `startswith` check. The string `filename` is supposed to be matched case-insensitively.

Reply:
Updated